packages/dev/core/src/Materials/greasedLinePluginMaterial.ts

```diff
@@ -294,6 +402,10 @@ export class GreasedLinePluginMaterial extends MaterialPluginBase {
             GreasedLinePluginMaterial._PrepareEmptyColorsTexture(_scene);
         }
 
+        this._engine.onDisposeObservable.add(() => {
+            GreasedLinePluginMaterial._EmptyColorsTexture?.dispose();

```

Comment:
Just one issue regarding disposing the texture on engine dispose - this is on the class level, and the texture is static, which means that if you generate 10 materials, this will be executed 10 times (and not 1). I guess setting it to null after disposing it will prevent that from happening and will be an acceptable solution.

Reply:
Yep.